x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/EqlSpec.java

```diff
@@ -17,6 +18,11 @@
     private String query;
     private long[] expectedEventIds;
 
+    // flag to dictate which modes are supported for the test
+    // at least one must be true
+    private boolean caseSensitive = false;
+    private boolean caseInsensitive = false;
```

Comment:
As on the previous issue, having a `Boolean` object should cover all the cases.
`TRUE` - sensitive, `FALSE` - insensitive, `null` - unspecified (apply both).